src/helper/filterTodos.ts

```diff
@@ -0,0 +1,21 @@
+import { LinksPath } from '../types/LinksPath';
+import { Todo } from '../types/Todo';
+
+export const filterTodos = (items: Todo[], filterBy: string) => {
+  if (filterBy === LinksPath.All) {
+    return items;
+  }
+
+  return items.filter(({ completed }) => {
+    switch (filterBy) {
+      case LinksPath.Active:
+        return !completed;
+
+      case LinksPath.Completed:
+        return completed;
+
+      default:
```

Comment:
Returning false in the default case will cause all items to be filtered out when an unknown filter is used. It's better to handle this case by either throwing an error or returning the original items.